src/components/Votes/VoteDisplay/VoteDisplay.scss

```diff
@@ -1,29 +1,139 @@
 @import "src/constants/style.scss";
 
+$vote-display-height: 42px;
+$vote-display-width: 114px;
+
 .vote-display {
+  height: $vote-display-height;
+  width: $vote-display-width;
+  color: $color-white;
+  background-color: $menu-icon-background-color--dark;
+  border-radius: calc($vote-display-height / 2);
   position: relative;
-  width: 85px;
+  border: 2px solid $menu-icon-background-color--dark;
 }
 
-.vote-display > * {
-  color: $color-white;
-  background-color: $color-backlog-blue;
-  vertical-align: bottom;
-  box-shadow: 0 8px 16px #0057ff25;
+.vote-display > svg {
+  position: absolute;
+  height: 100%;
+  background-color: $menu-icon-background-color--dark;
+  border-radius: 100%;
 }
-
-[theme="dark"] {
-  .vote-display > * {
-    background-color: $color-planning-pink;
-    box-shadow: 0 8px 16px #e2036025;
-  }
+[theme="dark"] .vote-display > svg {
+  background-color: $color-white;
+  color: $color-black;
 }
+
 .vote-display > span {
-  display: inline-block;
+  position: absolute;
+  right: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  padding-left: 38px;
```

Comment:
I guess your concern is that I have to apply a padding to the span. Wrapping them in a div wouldn't work because of the absolute positioning which is need to display the buttons between the icon and the text.

Reply:
IIRC I tested it and it was possible, but if you want to I can check again and show you